doc/lightning-offer.7.md

```diff
@@ -6,7 +6,7 @@ SYNOPSIS
 
 **(WARNING: experimental-offers only)**
 
-**offer** *amount* *description* [*issuer*] [*label*] [*quantity\_max*] [*absolute\_expiry*] [*recurrence*] [*recurrence\_base*] [*recurrence\_paywindow*] [*recurrence\_limit*] [*single\_use*]
+**offer** *amount* *description* [*issuer*] [*label*] [*quantity\_max*] [*absolute\_expiry*] [*recurrence*] [*recurrence\_base*] [*recurrence\_paywindow*] [*recurrence\_limit*] [*single\_use*] [*nodeid*]
```

Comment:
```suggestion
**offer** *amount* *description* [*issuer*] [*label*] [*quantity\_max*] [*absolute\_expiry*] [*recurrence*] [*recurrence\_base*] [*recurrence\_paywindow*] [*recurrence\_limit*] [*single\_use*] [*node\_id*]
```

Reply:
I think I copied it from `dev-rawrequest` which also uses `nodeid`, may want to fix that too? (I assume you're meaning change the code to be `node_id` not `nodeid` as well)

Reply:
For dev- commads it is fine I think, but for command that we build rust struct from code gen it is not IMHO otherwise we have a lot of bad naming around